esphome/components/mitsubishi/mitsubishi.cpp

```diff
@@ -367,6 +372,18 @@ bool MitsubishiClimate::on_receive(remote_base::RemoteReceiveData data) {
       break;
   }
 
+  switch (state_frame[14]) {
+    case MITSUBISHI_ECONOCOOL:
+      this->preset = climate::CLIMATE_PRESET_ECO;
+      break;
+    case MITSUBISHI_NIGHTMODE:
+      this->preset = climate::CLIMATE_PRESET_SLEEP;
+      break;
+    case MITSUBISHI_DEFAULTMODE:
```

Comment:
Hi @OlympusMonds ,
I am OK with adding nightmode as CLIMATE_PRESET_SLEEP. 
I am having a challenge with CLIMATE_PRESET_NONE. state_frame[14] and state_frame[15] are in the presets. My controls, remember mine are older units,  set 0x00 in these frames. I don't think MITSUBISHI_DEFAULTMODE  is a good way to evaluate that CLIMATE_PRESET_NONE is selected.

Reply:
No worries, I've removed that code, and it actually still works just fine.

Reply:
I think we’re good to merge. Thanks for working on this.